packages/http-error-handler/__tests__/index.js

```diff
@@ -99,6 +99,29 @@ test('It should be possible to pass a custom logger function', async (t) => {
   t.true(logger.calledWith(expectedError))
 })
 
+test('It should be possible to pass in headers with error', async (t) => {
+  const handler = middy(() => {
+    const error = createError(422, 'Unprocessable Entity')
+    error.headers = {
```

Comment:
You can also pass headers to the `createError` like this:
```ts
createError(424, { message: 'Unprocessable Entity', headers: { Location: 'http://exmaple.org/500' } });
```
but this also works.

Reply:
The `createError` function here is a slimmed down version built into middy. It only has the functionality needed for the middlewares.